### PR TITLE
[Fix][Zeta] Fix resource isolation not working on multi node

### PR DESCRIPTION
**Pull request by chenqianwen**
https://github.com/apache/seatunnel/issues/7469

### Purpose of this pull request

separated-cluster-deployment， When starting the worker, specify group=platform, and when submitting the job, specify tag_filter, which is also group=platform, but cannot be matched. 

### Check list

* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [x] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [x] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
cc @liunaijie as well.

**Comment by Hisoka-X** [comment]
Please add test case.

**Comment by liunaijie** [comment]
I check the code with my local env, i miss to submit this part of code, my bad. 
thanks @chenqianwen fix this issue.

The current test is only has 1 node, won't have any network transport. so not get exception.
Please also help update the test case, use 2 node as a cluster, thanks.

**Comment by chenqianwen** [comment]
I have add one test case named: WorkerTagClusterTest

**Review by Hisoka-X — DISMISSED** [review]
LGTM if ci passes. Thanks @chenqianwen !

**Comment by Hisoka-X** [comment]
Hi @chenqianwen , please retry the failed ci. Thanks.
image

File: seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/resourceIsolation/WorkerTagClusterTest.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+import com.hazelcast.config.Config;
+import com.hazelcast.instance.impl.HazelcastInstanceImpl;
+import com.hazelcast.spi.impl.NodeEngineImpl;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+/** 2024/08/26 17:36 yanggld */
```
Comment (Hisoka-X) [inline seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/resourceIsolation/WorkerTagClusterTest.java]:
```suggestion
```

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @chenqianwen

**Review by wuchunfu — APPROVED** [review]
LGTM, Looking forward to your next contribution, thanks

**Pull request merged**